kernel/model/bootstrap.go

```diff
@@ -88,6 +88,12 @@ func bootstrapModel() (*Model, error) {
 			}
 		}
 
+		for _, factory := range m.ModelFactories {
+			if err := factory.Build(m); err != nil {
+				return nil, fmt.Errorf("error executing factory [%s] (%w)", reflect.TypeOf(factory), err)
```

Comment:
been trying to standardize on `errors.Wrap` for this sort of thing.

`return nil, errors.Wrapf(err, "error executing factory [%s]", reflect.TypeOf(factory))`

Reply:
Good to know, I hadn't learned about errors.Wrap(f) yet. I've started converting things over, but there's a lot there, so I'll convert more opportunistically as I come across them.